libres/lib/enkf/gen_obs.cpp

```diff
@@ -244,12 +254,11 @@ gen_obs_type *gen_obs_alloc(const gen_data_config_type *data_config,
     if (error_covar_file != NULL) {
         FILE *stream = util_fopen(error_covar_file, "r");
 
-        obs->error_covar = matrix_alloc(obs->obs_size, obs->obs_size);
-        matrix_fscanf_data(obs->error_covar, false, stream);
+        obs->error_covar =
```

Comment:
Are there any tests using this functionality?

Reply:
I don't think so